msvc140/pch.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
```

Comment:
precompiled headers; only (simple) way to improve build speed with MSVC.
regardless, this header is only pulled in by the MSVC build and lives outside src/ (inside msvc140/).